tests/features/tmgmt/tmgmt_workbench.feature

```diff
@@ -158,4 +158,3 @@ Feature: TMGMT Workbench features
     And I select "Translator A" from "Translator"
     And I press "Submit to translator"
     Then I should not see the text "This content cannot be translated."
-    And I should see the text "Please end up the active translation process before creating a new request."
```

Comment:
We need to check if in the tmgmt_poetry this case is covered. Main reason for removing that step was to not to have a dependency from the tmgmt_poetry inside tmgmt_workbench.